### ISSUE TITLE
Farm job rework

### ISSUE THREAD

**Issue by MartinKlep** [klepek42/BETTER-TT-REALLIFE#29]
* Adjust some frustrating missions to make them less time intensive (e.g. reduce markes for the tractor missions)
* add new missions
* Offer more money in general, mission 8 and 9 should be compensated like truckerjobs and the other missions should be adjusted according to time needed
* Different wages for different missions
See following table for consideration

Nr. | Description  |  time in seconds  | factor (rounded up) | example wage
----| -------------| -------------| -------------| -------------
1 | "Erdbeeren pflücken" | 100 | 1,2 | 24
2 | "Heuregen" | 90  | 1 | 20
3 | "Düngen" | 560 | 6,3 | 126
4 | "Pflügen" | 515 | 5,8 | 116
5 | "Ernten" | 150 | 1,7 | 34
6 | "Dünger holen" | 330 | 3,7 | 74
7 | "Benzin holen" | 140 | 1,6 | 32
8 | "Apfelkuchen" | 220 | 2,5 | 50
9 | "Holz abholen" | 225  | 2,5 | 50

**Comment by klepek42** [comment]
Good idea! With the given factors you could adjust the table for this job in "pay_skill_sys.lua" with additional if-statements.
